### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by JosephSC0121**
Pr para página en producción

### CODE REVIEW

**Review by Camilo-Camargo — APPROVED** [review]

File: src/lib/directus.ts
Comment (CamiloArias12) [inline src/lib/directus.ts:1]: eliminar el console.log()

File: public/favicon.svg
Comment (Alejandro-Suarez-Rincon) [inline public/favicon.svg:1]: Este no seria bueno eliminarlo?

File: src/lib/directus.ts
Comment (CamiloArias12) [inline src/lib/directus.ts:1]: eliminar console.log()

**Review by CamiloArias12 — CHANGES_REQUESTED** [review]

File: src/lib/directus.ts
Comment (Camilo-Camargo) [reply src/lib/directus.ts:1]: https://github.com/OwlByTech/owlbytech/pull/3/commits/22a39e3e347c183fcae1f8dc058535c3118cf741

Comment (Camilo-Camargo) [reply src/lib/directus.ts:1]: https://github.com/OwlByTech/owlbytech/pull/3/commits/22a39e3e347c183fcae1f8dc058535c3118cf741

File: public/favicon.svg
Comment (Camilo-Camargo) [reply public/favicon.svg:1]: https://github.com/OwlByTech/owlbytech/pull/3/commits/7cdc7c576937ee54f53a98d9557162b78fe64023

**Review by CamiloArias12 — APPROVED** [review]

**Review by Camilo-Camargo — APPROVED** [review]

**Review by Alejandro-Suarez-Rincon — APPROVED** [review]

**Review by Alejandro-Suarez-Rincon — APPROVED** [review]

**Pull request merged**